### DISCUSSION TITLE
Fenicsx Test avec demo_poisson.py

### DISCUSSION THREAD

**Discussion by HunchoFR** [msolides-2023/MU5MES01-2023#6] [Q&A]
Bonsoir,

J'ai essayé d'installer fenics aujourd'hui. 
J'ai du rajouter des # pour mettre certaines parties du fenicsx-0.6.0.yml en commentaires
![Capture](https://github.com/msolides-2023/MU5MES01-2023/assets/54946501/bf5b6f7e-7b3b-4e4c-a988-312e81d693eb)
mais l'installation semble avoir aboutie.
Ensuite j'ai installé chaque dépendances à la main avec la commande pip install et sudo install "dependance" et j'ai fait de même pour la section pip du fichier yml.
(pip install pyaml ; pip install jupyterlab;pip install numba etc etc ...)

Lorsque j'essaie de tester en utilisant le démo "poisson.py", je rencontre une erreur.
![ubuntu](https://github.com/msolides-2023/MU5MES01-2023/assets/54946501/e4bcee41-860d-4b20-b0a5-2c2c4e304925)

Pourtant j'ai remarqué que les dossiers associés ont été créés au moment de la commande dans le fichier out_poisson.
![fichiers](https://github.com/msolides-2023/MU5MES01-2023/assets/54946501/a045928d-cb16-4e0e-9c3d-e6801c4641a5)

J'ai aussi testé jupyter, qui semble fonctionner mais j'ai des erreurs lors de l'execution de certaines cellules
![jupyter erreur](https://github.com/msolides-2023/MU5MES01-2023/assets/54946501/54b04aec-4336-4dfb-b1e7-6b73e40de484)

Je vais peut etre essayer la méthode avec mamba mais je ne sais pas si vous avez des pistes pour résoudre ces problèmes ?
Merci d'avance pour votre aide.

**Comment by cmaurini** [comment]
Merci pour le travail et le rapport complet.

Essayez d'ajouter après l'import de pyvista:
```
pyvista.set_jupyter_backend("static")
pyvista.start_xvfb()
```

Merci de confirmer si le problème est résolu
